Rakefile

```diff
@@ -19,7 +19,7 @@ file "head-wasm32-unknown-wasi-full-js" do
   version = JSON.parse(File.read("package.json"))["dependencies"]["ruby-head-wasm-wasi"][1..]
   filename = "ruby-head-wasm32-unknown-wasi-full-js.tar.gz"
 
-  `curl -LO https://github.com/ruby/ruby.wasm/releases/download/ruby-head-wasm-wasi-#{version}/#{filename}`
+  `curl -LO https://github.com/ruby/ruby.wasm/releases/download/ruby-head-wasm-wasi-#{version}/ruby-head-wasm32-unknown-wasi-full-js.tar.gz`
```

Comment:
I think this line is equivalent and can be reverted

Reply:
You're right, It was just a missing test.